### ISSUE TITLE
TypeError: (0 , decamelize_1.default) is not a function

### ISSUE THREAD

**Issue by fredericosilva** [mirego/accent#379]
After updating the package I started to get this error:

```sh
npx accent sync --dry-run
Fetch config in accent.json... API Client ✓

Syncing sources → en  ✓

    TypeError: (0 , decamelize_1.default) is not a function
```

accent-cli@0.12 is latest working version

js bundle references a ```decamelize_1.default``` that doesn't exist

https://unpkg.com/accent-cli@0.14.1/lib/services/formatters/hook-runner.js

```javascript
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
// Vendor
const chalk = require("chalk");
const decamelize_1 = require("decamelize");
const capitalizeFirstLetter = (str) => str.charAt(0).toUpperCase() + str.slice(1);
class HookRunnerFomatter {
    log(name, commands) {
        const operation = capitalizeFirstLetter((0, decamelize_1.default)(name, { separator: ' ' }));
        console.log(chalk.yellow('➤ '), chalk.bold(chalk.yellow(`${operation}:`)));
        commands.forEach((command) => {
            console.log('  ', chalk.yellow(command));
        });
        console.log('');
    }
}
exports.default = HookRunnerFomatter;
```

Let me know if you need more information.

**Comment by simonprev** [comment]
Right, the dependency was removed in a refactor but I did not test the hooks feature! Will be fixed in the next release!

**Comment by simonprev** [comment]
Fixed in `0.14.2`: https://www.npmjs.com/package/accent-cli/v/0.14.2
